src/components/forms/SelectField/SelectField.js

```diff
@@ -63,8 +69,9 @@ const SelectField = ({
         )}
         {...props}
       />
-      {description && <FormFieldDescription invalid={invalid}>{description}</FormFieldDescription>}
-    </>
+      {description && <FormFieldDescription>{description}</FormFieldDescription>}
```

Comment:
For the open-forms theme, the padding above the description of the select is not quite consistent with the other components:
![Screenshot from 2023-05-26 11-32-07](https://github.com/open-formulieren/open-forms-sdk/assets/19154114/816e2690-a566-47b0-8451-d7299bb01c9f)

Reply:
yeah the `Description` styling itself is also not consistent yet, I was planning to tackle that in another issue/PR and wanted to focus this on the validation errors (styling).